test/system/065-cp.bats

```diff
@@ -149,7 +149,7 @@ load helpers
     is "$output" "" "output from podman cp 1"
 
     run_podman cp --pause=false $srcdir/$rand_filename2 cpcontainer:/tmp/d2/x/
-    is "$output" "" "output from podman cp 3"
+    is "$output" "Error: failed to get stat of dest path .*stat.* no such file or directory" "cp returns error no such file or directory"
```

Comment:
You need to change the expected exit status, e.g.:
```
run_podman 12345 cp --pause ...
```
...where 12345 is the expected exit status of the cp command (probably 1, but may be something else, see test logs)

Reply:
@edsantiago  thanks. :slightly_smiling_face:  Do I still need keep this current change?

Reply:
The check for "Error"? Yes.

Reply:
:ok